test/e2e/apps/deployment.go

```diff
@@ -271,9 +271,8 @@ var _ = SIGDescribe("Deployment", func() {
 					"spec": map[string]interface{}{
 						"TerminationGracePeriodSeconds": &zero,
 						"containers": [1]map[string]interface{}{{
-							"name":    testDeploymentName,
-							"image":   testDeploymentPatchImage,
-							"command": []string{"/bin/sleep", "100000"},
```

Comment:
this is a patch request, do you have to set `"command": []string{}` explicitly to remove the existing command?

Reply:
from my testing, remove the command line can make it work, the pod will be running successfully.